### PR TITLE
[Added TS] Add (Neox Scanlator) Presence

### PR DESCRIPTION
**Pull request by Nanashinn**
image
image
image
image
image

### CODE REVIEW

**Comment by Bas950** [comment]
On Hold - We are changing some rules regarding presence creation, we will update you soon.

**Comment by Bas950** [comment]
Please code your presence in TypeScript (TS).
For more info see:
https://discordapp.com/channels/493130730549805057/607524579874832446/704792826025869393

File: Neox Scanlator/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "dist"
+    }
+  }
```
Comment (Bas950) [inline Neox Scanlator/tsconfig.json:6]: https://docs.premid.app/en/dev/presence#filling-in-the-tsconfigjson-file

File: Neox Scanlator/presence.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    let opcaoLeitor: any;
+    let tipoObra: any;
+    let generoObra: any;
+    let spanObra: any;
+    let nomeObra: any;
+    let nomeObraLeitor: any;
+    let capituloLeitor: any;
+    let seasonLeitor: any;
+    let paginaLeitor: any;
+    let postagemData: any;
```
Comment (Bas950) [inline Neox Scanlator/presence.ts]: use commas:
Let PesquisaTexto, UsuarioTexto, etc.
They will all be any automatically that way.

File: Neox Scanlator/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+     },
+     "service": "Neox Scanlator",
+     "description": {
+       "en": "One of the giants fansubs in Brazil where they translate and distribute to the public, also known for translating Solo Leveling and other famous manga.",
+       "pt_BR": "Uma das gigantes fansubs do Brasil, onde traduzem e libera ao público, também conhecida por traduzir obras como Solo Leveling além de outras obras famosas.",
+       "pt": "Uma das gigantes fansubs do Brasil, onde traduzem e libera ao público, também conhecida por traduzir obras como Solo Leveling além de outras obras famosas."
+     },
+     "url": "neoxscan.com",
+     "version": "1.0.0",
+     "logo": "https://i.imgur.com/foWmeiz.png",
```
Comment (frozaum) [inline Neox Scanlator/dist/metadata.json:14]: logo looks stretched, please fix that

**Pull request merged**